arch/arm/core/aarch32/Kconfig.mcux

```diff
@@ -0,0 +1,5 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config CPU_CORTEX_M
```

Comment:
Why is this needed since you have selected `DCACHE` in the `Kconfig.soc` file

Reply:
gone